documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -0,0 +1,393 @@
+ | controller.rest.tlsKeyStorePasswordFile | `/etc/secrets/controller01.server.jks.password`<sup>1</sup> |
+ | controller.zk.secureConnection | `false`<sup>2</sup> |
+ | controller.zk.tlsTrustStoreFile | `/etc/secrets/client.truststore.jks` |
+ | controller.zk.tlsTrustStorePasswordFile | `/etc/secrets/client.truststore.jks.password` |
+ | controller.auth.enabled | `true` |
+ | controller.auth.userPasswordFile<sup>3</sup> | `/etc/secrets/password-auth-handler.input` |
+ | controller.auth.tokenSigningKey | `a-secret-value` |
+
+ <sup>1</sup>: This and other `.password` files are text files containing the password for the corresponding store.
+
+ <sup>2</sup>: The assumption is that Zookeeper TLS is disabled. You may enable it and specify the corresponding client-side TLS configuration properties via the `controller.zk.*` properties.
+
+ <sup>3</sup>: This configuration property is required when using the default Password Auth Handler only.
+
+**Segment Store**
+
+Segment store supports configuration via a properties file (`config.properties`) or JVM system properties. The table below lists its TLS and auth parameters and sample values. For a detailed discription of these parameters refer to [this](https://github.com/pravega/pravega/blob/master/documentation/src/docs/security/pravega-security-configurations.md) document.
+
+ | Configuration Parameter| Sample Value |
+ |:-----------------------:|:-------------|
+ | pravegaservice.enableTls | `true` |
+ | pravegaservice.certFile | `/etc/secrets/segmentstore01.pem` |
+ | pravegaservice.keyFile | `/etc/secrets/segmentstore01.key.pem` |
+ | pravegaservice.keyFile | `/etc/secrets/segmentstore01.key.pem` |
+ | pravegaservice.secureZK | `false`<sup>2</sup> |
+ | pravegaservice.zkTrustStore | `/etc/secrets/client.truststore.jks` |
+ | pravegaservice.zkTrustStorePasswordPath | `/etc/secrets/client.truststore.jks.password` |
+ | autoScale.tlsEnabled | `true` |
+ | autoScale.tlsCertFile | `/etc/secrets/segmentstore01.key.pem` |
+ | autoScale.authEnabled | `true` |
+ | autoScale.tokenSigningKey | `a-secret-value` <sup>1</sup>|
+ | autoScale.validateHostName | `true` |
+
+<sup>1</sup>: The assumption is that Zookeeper TLS is disabled. You may enable it and specify the corresponding client-side TLS configuration properties via the `pravegaservice.zk.*` properties.
+
+<sup>2</sup>: The secret value you use here must match the same value used for other Controller and Segment Store services.
+
+### Configuring TLS and Credentials on the Client Side
+
+After you have enabled TLS and Auth and configured them for server-side services, you need to update any existing client
+applications to use TLS and supply their credentials. This sub-section described how to do that.
+
+For TLS, you'll need to establish trust for the servers' certificates on the client sde. There are multiple ways of
+doing that:
+
+  1. Supplying the client library with the certificate of the trusted CA that has signed the servers' certificates.
```

Comment:
servers` ...typo

Reply:
Since Pravega can be deployed on multiple servers, I was using the plural form of the term "server" here. So, servers' is correct.